cucumber-core.gemspec

```diff
@@ -22,8 +22,8 @@ Gem::Specification.new do |s|
                     'source_code_uri' => 'https://github.com/cucumber/cucumber-ruby-core',
                   }
 
-  s.add_dependency 'cucumber-gherkin', '~> 18.1', '>= 18.1.0'
-  s.add_dependency 'cucumber-messages', '~> 15.0', '>= 15.0.0'
+  s.add_dependency 'cucumber-gherkin', '~> 19.0', '>= 19.0.1'
```

Comment:
I've released 19.0.2

Reply:
Ok :D

I'll take a look on it tomorrow 👍